content/docs/next-release/sampling.md

```diff
@@ -21,7 +21,7 @@ Adaptive sampler is a composite sampler that combines two functions:
   * It makes sampling decisions on a per-operation basis, i.e. based on {{< tip "span" >}} operation name. This is especially useful in the API services whose endpoints may have very different traffic volumes and using a single probabilistic sampler for the whole service might starve (never sample) some of the low QPS endpoints.
   * It supports a minimum guaranteed rate of sampling, such as always allowing up to N {{< tip "traces" "trace" >}} per seconds and then sampling anything above that with a certain probability (everything is per-operation, not per-service).
 
-Per-operation parameters can be configured statically or pulled periodically from Jaeger backend with the help of Remote sampler. Adaptive sampler is designed to work with the upcoming [Adaptive Sampling](https://github.com/jaegertracing/jaeger/issues/365) feature of the Jaeger backend.
+Per-operation parameters can be configured statically or pulled periodically from Jaeger backend with the help of Remote sampler. In order to use adaptive sampling set the environment variable `SAMPLING_TYPE=adaptive` on the collector. Currently this is only available when paired with a Cassandra `SPAN_STORAGE_TYPE`. If you're just getting started please review `sampling.initial-sampling-probability` and `sampling.target-samples-per-second`, and see the [cli parameters](/docs/latest/cli/#jaeger-collector-adaptive) for details on advanced configuration.
```

Comment:
I don't mind merging like this, but I think we need a bit more documentation explaining how the thing actually works. I would also separate the section into two parts, one (existing) describing just the per-operation functionality (which is not technically "adaptive"), and the actual new adaptive part.

Reply:
The description of static sampling is basically here:

https://www.jaegertracing.io/docs/1.26/sampling/#collector-sampling-configuration

but agree the way this doc is organized makes things a bit confusing. We should probably move the adaptive sampling under "collector sampling configuration" and have two sections there (one for adaptive and one for static). I will update this PR with some structural changes.

Reply:
changes made